tests/storage/mailbox_test.py

```diff
@@ -357,13 +421,15 @@ def reply_msg_callback(vol_data):
                         # check that SPM inbox was cleared
                         if f.read(sm.MAILBOX_SIZE) == sm.EMPTYMAILBOX:
                             break
-                    assert time.time() < deadline, "Timeout clearing SPM inbox"
-                    time.sleep(0.1)
+                    if time.monotonic() < deadline:
```

Comment:
Also just for my information. You changed to `monotonic()` to protect from summer --> winter --> summer clock transitions situations?
I thought `time.time()` is used widely and is enough.

Reply:
time.time() is not affected by daylight saving changes, but it can go backwards in the system time changes. It is ok for tests but there is no reason to use it in python 3 since we have time.monotonic(). All code checking for timeouts should use time.monotonic().

Regarding the check - yes, it should should be:

    if time.monotonic() > deadline:
        fail...